### PR TITLE
Only 15u of thermite is needed to burn walls now

### PR DESCRIPTION
**Pull request by ghost**
:cl: Thefastfoodguy
tweak: Only 15u of thermite is needed to burn walls now
/:cl:

### CODE REVIEW

**Comment by tortellinitony** [comment]
I don't see at all why thermite needs a buff since it's so easy to make. THis means that you could thermite 8 walls with a beaker instead of 2...

**Comment by ghost** [comment]
@tortellinitony you didn't need 50u to burn a wall down before /tg/ memes refactored thermite, you could easily use 20-30u before

**Comment by tortellinitony** [comment]
That doesn't answer my concerns, though.

**Comment by KayeArray** [comment]
Just re-balance it to exactly what it was before and you have my +1

**Comment by ghost** [comment]
@tortellinitony People don't know that 50u is required, and I constantly get mhelped "why my thermite no work"

File: code/datums/components/thermite.dm [hunk 1]
```diff
@@ -41,7 +41,7 @@
 
 	playsound(master, 'sound/items/welder.ogg', 100, 1)
 
-	if(amount >= 50)
+	if(amount >= 15) //changed from 50 to 15 by hippiememes
```
Comment (KayeArray) [inline code/datums/components/thermite.dm]: can you not just mirror this proc

Comment (Chnkr) [reply code/datums/components/thermite.dm]: Please mark your changes. https://github.com/HippieStation/HippieStation/blob/master/hippiestation/README.md#please-mark-your-changes

Comment (ghost) [inline code/datums/components/thermite.dm]: You've made changes to /tg/ files, please mark these changes.

**All modifications to non-hippie files should be marked.**

- Multi line changes start with `// hippie start` and end with `// hippie end`
- You can put a messages with a change if it isn't obvious, like this: `// hippie start - reason`
  - Should generally be about the reason the change was made, what it was before, or what the change is
  - Multi-line messages should start with `// hippie start` and use `/* Multi line message here */` for the message itself
- Single line changes should have `// hippie` or `// hippie - reason`

If you need to mirror a file, or function into a hippie-specific file, please leave behind a comment stating where it went.

```
// hippie start - Mirrored this function in <file> for <reason>
bunch of shitcode here
// hippie stop
```

Once you mirror a file, please follow the above for marking your changes, this way we know what needs to be updated when a file has been mirrored.

**Review by ghost — DISMISSED** [review]

**Comment by dreamcrab** [comment]
15 seems too low? im not sure i care if thats how it used to be

**Pull request closed without merging**